<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>webget</class>
 <widget class="QDialog" name="webget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Execute</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Image URL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Update delay</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>371</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="url1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="url2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="url3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="url4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="url5"/>
    </item>
    <item>
     <widget class="QLineEdit" name="url6"/>
    </item>
    <item>
     <widget class="QLineEdit" name="url7"/>
    </item>
    <item>
     <widget class="QLineEdit" name="url8"/>
    </item>
    <item>
     <widget class="QLineEdit" name="url9"/>
    </item>
    <item>
     <widget class="QLineEdit" name="url10"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>111</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QDoubleSpinBox" name="millis1">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.003000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="millis2">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.003000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="millis3">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.003000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="millis4">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.003000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="millis5">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.003000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="millis6">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.003000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="millis7">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.003000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="millis8">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.003000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="millis9">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.003000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="millis10">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.003000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
